types/ali-oss/index.d.ts

```diff
@@ -180,6 +181,7 @@ declare namespace OSS {
         name: string;
         data: object;
         res: NormalSuccessResponse;
+        headers?: object;
```

Comment:
This line @imhele

Reply:
https://github.com/ali-sdk/ali-oss/blob/master/lib/object.js#L76

Reply:
I think you should add the headers type in the `PutObjectOptions ` interface, not the `PutObjectResult ` interface
@imhele

Reply:
oh, thanks